docs/modalcard.md

```diff
@@ -36,54 +42,84 @@ $assetManager->registerMany([
-`options(array $value)` | HTML attributes for the widget container tag. | [`class` => `modal`]
-`contentOptions(array $value)` | HTML attributes for the widget content tag. | [`class` => `modal-card`]
-`headerOptions(array $value)` | HTML attributes for the widget header tag. | [`class` => `modal-card-head`]
-`title(string $value)` | The title content in the modal window. | `''`
-`titleOptions(array $value)` | HTML attributes for the widget title tag. | [`class` => `modal-card-title`]
-`bodyOptions(array $value)` | HTML attributes for the widget body tag.| [`class` => `modal-card-body`]
-`footer(string $value)` | The footer content in the modal window. | `''`
-`footerOptions(array $value)` | HTML attributes for the widget footer tag. | [`class` => `modal-card-foot`]
-`closeButtonOptions(array $value)` | HTML attributes for the widget close button tag. | [`class` => `delete`, `aria-label` => `close`]
-`closeButtonSize(string $value)` | Size close button. | `is-small`, `is-medium`, `is-large`
-`toggleButtonLabel(string $value)` | Toggle button label, | `Toggle button`
-`toggleButtonSize(string $value)` | Size toggle button. | `is-small`, `is-medium`, `is-large`
-`toggleButtonColor(string $value)` | Toggle button color. | `is-primary`, `is-link`, `is-info`, `is-success`, `is-warning`, `is-danger`
-`toggleButtonOptions(array $value)` |  HTML attributes for the widget toogle button tag. | [`class` => `button`, `aria-haspopup` => `true`]
-`withoutCloseButton()` | Disable close button. | `false`
-`withoutToggleButton()` | Disable toggle button. | `false`
+`backgroundClass(string $value)` | The class for the modal background. | `modal-background`
+`bodyAttributes(array $value)` | HTML attributes for the body tag.| []
+`bodyClass(string $value)` | The class for the modal body. | `modal-card-body`
+`buttonClass(string $value)` | The class for the modal button. | `button modal-button`
+`cardAttributes(array $value)` | HTML attributes for the card. | []
+`closeButtonAttributes(array $value)` | HTML attributes for the close button. | []
+`closeButtonClass(string $value)` | The class for the close button. | `button delete`
+`closeButtonSize(string $value)` | Size close button. Options available are: (`ModalCard::SIZE_SMALL`, `ModalCard::SIZE_MEDIUM`, `ModalCard::SIZE_LARGE`). | Default setting empty normal.
+`contentClass(string $value)` | The class for the modal card content. | `modal-card-content`
+`footClass(string $value)` | The class for the modal card foot. | `modal-card-foot`
+`footer(string $value)` | The footer content. | `''`
+`footerAttributes(array $value)` | HTML attributes for the footer. | []
+`headClass(string $value)` | The class for the modal card head. | `modal-card-head`
+`headerAttributes(array $value)` | HTML attributes for the header. | []
+`id(string $value)` | Widget ID. | `''`
+`modalCardClass(string $value)` | The class for the modal card. | `modal-card`
```

Comment:
Can we remove "modalCard" prefix?

Reply:
It's the CSS Class of the `ModalCard`, as I call it CardCssClass ?

Reply:
Yes. That's better name 👍

Reply:
done.